scripts/env/winesapos-env-defaults.sh

```diff
@@ -7,7 +7,7 @@ export \
   WINESAPOS_DISTRO="${WINESAPOS_DISTRO:-arch}" \
   WINESAPOS_DISTRO_DETECTED="${winesapos_distro_detected}" \
   WINESAPOS_ENABLE_TESTING_REPO="${WINESAPOS_ENABLE_TESTING_REPO:-false}" \
-  WINESAPOS_DE="${WINESAPOS_DE:-plasma}" \
+  WINESAPOS_DE="${WINESAPOS_DE:-cinnamon}" \
```

Comment:
I'm not ready to switch desktop environments yet. The goal is to do that later in 2025 along with other large changes to winesapOS. One of those large changes is dropping Xorg support. Cinnamon has initial Wayland support but it's still very broken right now.

Reply:
Alright